app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @ingredients.each do |i| %>
+    <%= i.name %>
+    <%= i.cost %>
+<% end %>
```

Comment:
This is great but your code is missing the required HTML. This is still an HTML file, so HTML is expected.

```erb
<% @ingredients.each do |i| %>
  <p><%= i.name %></p>
  <p><%= i.cost %></p>
<% end %>
```

These are just some HTML suggestions, you can use any others that make sense, like a list.